geode-core/src/test/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderEventProcessorJUnitTest.java

```diff
@@ -88,7 +90,7 @@ public void validateBatchConflationWithBatchContainingDuplicateConflatableEvents
     originalEvents.add(ParallelGatewaySenderHelper.createGatewaySenderEvent(lr, Operation.UPDATE,
         "Object_13964", lastUpdateValue, lastUpdateSequenceId, lastUpdateShadowKey));
     originalEvents.add(ParallelGatewaySenderHelper.createGatewaySenderEvent(lr, Operation.DESTROY,
-        "Object_13964", null, 104, 28274));
+        "Object_13964", null, 105, 28274));
```

Comment:
Sorry it isn't obvious to me why this was changed from 104 to 105.  Do you mind explaining?